docs/storing-querying/storage-options/index.md

```diff
@@ -40,12 +46,41 @@ Data warehouses and data lakes are primary destinations for Snowplow data. For o
 <AzureExperimental/>
 
 | Destination | Type | Loader application | Status |
-| --- | --- | --- |
+| --- | --- | --- | --- |
 | Snowflake | Micro-batching | [Snowplow RDB Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/index.md) | Early release |
+| Databricks | Micro-batching<br/>_(via a [data lake](#data-lake-loaders))_ | LAKELOADERLINK | Early release |
+| Synapse Analytics | Micro-batching<br/>_(via a [data lake](#data-lake-loaders))_ | LAKELOADERLINK | Early release |
 
   </TabItem>
 </Tabs>
 
-### Data lake loaders
+## Data lake loaders
+
+All lake loaders are micro-batching.
+
+<Tabs groupId="cloud" queryString>
+  <TabItem value="aws" label="AWS" default>
+
+| Lake | Format | Compatibility | Loader application | Status |
+| --- | --- | --- | --- | --- |
+| S3 | TSV | Amazon Athena | [S3 Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/s3-loader/index.md) | Production-ready |
+
+  </TabItem>
+  <TabItem value="gcp" label="GCP">
+
+| Lake | Format | Compatibility | Loader application | Status |
+| --- | --- | --- | --- | --- |
+| GCS | Delta | Databricks | LAKELOADERLINK | Early release |
+| GCS | TSV | BigQuery | [GCS Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/google-cloud-storage-loader/index.md) | Only recommended for [failed events](/docs/understanding-your-pipeline/failed-events/index.md) |
```

Comment:
Failed events are JSON format, not TSV.

The GCS loader is similar to the S3 loader, in that it doesn't care about the format.  But if we're recommending it only for failed events, then JSON is the only sensible thing to put in the second column.